package.json

```
@@ -3,7 +3,7 @@
   "version": "0.13.1",
   "description": "ESLint plugin to check Node.js dependencies.",
   "engines": {
-    "node": ">=14.17.0"
+    "node": ">=18.0.0"
```

It seems that we can use v12 of npm-package-arg if we tighten the requirements.

```suggestion
    "node": "^18.17.0 || >=20.5.0"
```

---

Oh, sorry, should we? I'm both fine with that. Another PR on the way.